pom.xml

```diff
@@ -97,7 +97,7 @@
     <jetty-servlet25.version>7.6.21.v20160908</jetty-servlet25.version>
     <resteasy.version>3.14.0.Final</resteasy.version>
 
-    <kafka.version>3.0.0</kafka.version>
+    <kafka.version>3.2.0</kafka.version>
```

Comment:
@frosiere do you downgrade to `3.2.0` because of `kafka-junit` dependency?

Reply:
Yes, I had test issues when using 3.2.1 and there is no version of kafka-junit supporting Kafka 3.2.1 yet.
Hope the build is fixed now.